### PR TITLE
fix(experience): add empty scopes fallback layout

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Add empty scopes fallback layout. This is a fallback layout of the consent page for corner cases like:
- no user scopes and resource scopes are requested e.g. only 'openid' and 'offline_access';
- all the requested scopes are granted previously. 

image

## Testing

test locally

## Checklist

- [ ] ~`.changeset`~
- [ ] ~unit tests~
- [ ] ~integration tests~
- [ ] ~necessary TSDoc comments~

### CODE REVIEW

**Review by charIeszhao — APPROVED** [review]
Approved with comments

File: packages/experience/src/pages/Consent/ScopesListCard/index.tsx [hunk 1]
```diff
@@ -92,6 +89,22 @@ const ScopesListCard = ({
...
+        >
+          {t('description.authorize_agreement', { name: appName })}
+        </Trans>
+      </div>
+    ) : null;
+  }
+
   return (
     <div className={className}>
       <div className={styles.title}>{t('description.request_permission', { name: appName })}</div>
```
[the remark this answers is not in the document]
Comment (simeng-li) [reply packages/experience/src/pages/Consent/ScopesListCard/index.tsx:110]: The layout are different.  If no scopes
1. we don't render the section title
2. terms link content won't be wrapped in a wrapper.
